db_facts/db_info.py

```diff
@@ -10,7 +10,10 @@
 
 
 def db(db_name: DBName, dbcli_config: DBCLIConfig = None) -> DBFacts:
-    """Get connection info for specified database."""
+    """Get connection info for specified database.
+
+    :param db_name: Alias for the particular database endpoint and account to connect to.  ['a','b','c'] corresponds to 'a-b-c' on the db-facts command-line.
```

Comment:
[flake8] E501 line too long (157 > 100 characters)

Reply:
I punted on this one; I couldn't figure out how to add a continuation to a directive in RST.  Maybe it's worth flipping over to google format docstrings?  Not sure if this is solvable there or not, just can't remember if you had a take on that.  I think the hypermodern series uses google format instead of RST.